src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdaterTest.java

```diff
@@ -4,55 +4,58 @@
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
For now, pls remove Ignored test cases. As we have to give unnecessary justification to release team why this test been ignored.